biorbd_optim/limits/path_conditions.py

```diff
@@ -242,6 +261,32 @@ def concatenate(self, other):
         self.min = PathCondition(np.concatenate((self.min, other.min)), interpolation=self.min.type)
         self.max = PathCondition(np.concatenate((self.max, other.max)), interpolation=self.max.type)
 
+        self.type = self.min.type
+        self.t = self.min.t
+        self.extra_params = self.min.extra_params
+        self.nb_shooting = self.min.nb_shooting
+
+    def __getitem__(self, slice_list):
+        if isinstance(slice_list, slice):
+            t = self.min.t
+            param = self.extra_params
+            interpolation = self.type
+            if interpolation == InterpolationType.CUSTOM:
+                min_bound = self.min.custom_function
+                max_bound = self.max.custom_function
+            else:
+                min_bound = np.array(self.min[slice_list.start : slice_list.stop : slice_list.step])
+                max_bound = np.array(self.max[slice_list.start : slice_list.stop : slice_list.step])
+            bounds_sliced = Bounds(
+                min_bound=min_bound, max_bound=max_bound, interpolation=interpolation, slice_list=slice_list, t=t, **param
+            )
+            # TODO: Verify if it is ok that slice_list arg sent is used only if it is a custom type (otherwise, slice_list is used before calling Bounds constructor
```

Comment:
That is why it should not be sent in the constructor. What if someone add a slice_list argument to the constructor for a non-custom type? Should we slice? Should we not? 
This slicing thing is a private concern and therefore a user should not be aware of this